client/beefy/rpc/src/lib.rs

```diff
@@ -62,26 +99,55 @@ pub trait BeefyApi<Notification, Hash> {
 		metadata: Option<Self::Metadata>,
 		id: SubscriptionId,
 	) -> jsonrpc_core::Result<bool>;
+
+	/// Returns header of the latest BEEFY finalized block as seen by this client.
+	///
+	/// The latest BEEFY block might not be available if the BEEFY gadget is not running
+	/// in the network or if the client is still initializing or syncing with the network.
+	/// In such case an error would be returned.
+	#[rpc(name = "beefy_getFinalizedHead")]
+	fn latest_finalized(&self) -> FutureResult<Header>;
```

Comment:
I reviewed this file last so I already alluded to this in another comment but I think this could instead just return a hash. `chain_getFinalisedHead` also returns just a hash (although the pubsub subscription returns headers).

Reply:
Indeed, hash was also discussed as an option in [this comment](https://github.com/paritytech/substrate/pull/10568#discussion_r777457540).

I currently have little to no real world experience on how these RPCs are used in the wild, so I will defer to your expertise on which is more useful between block _header_ or block _hash_.

At least _hash_ would be consistent with `chain_getFinalisedHead` RPC..

Reply:
Changed to provide _hash_.